src/languages/java.md

```diff
@@ -0,0 +1,3 @@
+# Java
+
+Java support is in Alpha and not generally available. Contact sales [https://platform.sh/contact](https://platform.sh/contact) if you are interested in running Java based applications on Platform.sh
```

Comment:
We are not even listing the container name. This is so we can start talking with people. That's all. This is close alpha.